functests/performance/performance.go

```diff
@@ -170,8 +173,60 @@ var _ = Describe("performance", func() {
 			execSysctlOnWorkers(workerRTNodes, sysctlMap)
 		})
 	})
+
+	Describe("Updating parameters in performance profile", func() {
```

Comment:
Let's add the rfe id while we are at it.

Reply:
added [rfe_id:28761]